terraform/module-cloud-watch/main.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_cloudwatch_event_rule" "payment_reconciler_daily" {
+  name        = "payment_reconciler_daily"
+  description = "Call payment Reconciler lambda"
+  schedule_expression ="cron(0 12 * * ? *)"
+}
+
+resource "aws_cloudwatch_event_target" "call_payment_reconciler_lambda_everyday" {
+    rule = "${aws_cloudwatch_event_rule.payment_reconciler_daily.name}"
+    target_id = "payment_reconciler"
```

Comment:
This should be a unique id. If there is a potential for a deployment per environment then this might fail. I would rename from `"payment_reconciler"` to `"${var.project-name}-${var.env}"`